files/haproxy.init

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh
+#
+# chkconfig: - 85 15
+# description: HA-Proxy is a TCP/HTTP reverse proxy which is particularly suited \
+#              for high availability environments.
+# processname: haproxy
+# config: /etc/haproxy/haproxy.cfg
+# pidfile: /var/run/haproxy.pid
+
+# Script Author: Simon Matter <simon.matter@invoca.ch>
+# Version: 2004060600
```

Comment:
I've decided to remove the haproxy.init file completely and instead add a parameter that let people specify an init file to be installed.  If someone is using haproxy::instance{}, they are going to have a custom init.d/haproxy-$GROUP that they want installed and if I include a sample it will just confuse them.  I could include an example in the future and make it the default to this define.